### PR TITLE
[main][test] smoke test deposit withdraw

### PR DESCRIPTION
**Pull request by chenciao8**
Smoke test on
- deposit / withdraw hlp
- deposit / withdraw collateral
- open / close position

### CODE REVIEW

File: test/fork/smoke-test/Smoke_Liquidity.sol [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+      bytes32[] memory _priceUpdateCalldata,
+      bytes32[] memory _publishTimeUpdateCalldata
+    ) = ecoPythBuilder.build(data);
+
+    vm.prank(address(0xF1235511e36f2F4D578555218c41fe1B1B5dcc1E));
+    liquidityHandler.executeOrder(_latestOrderIndex, payable(ALICE), _priceUpdateCalldata, _publishTimeUpdateCalldata, _minPublishTime, keccak256("someEncodedVaas"));
+    
+    assertApproxEqRel(
+      ForkEnv.usdc_e.balanceOf(ALICE),
+      10 * calculator.getAUME30(false) * 1e18 * 1e6 / 1e30 / hlp.totalSupply(),
```
Comment (john-xina-p88) [inline test/fork/smoke-test/Smoke_Liquidity.sol:106]:
```suggestion
      10 * calculator.getAUME30(false) * 1e18 * 1e6 / 1e30 / hlp.totalSupply(),
```
make this more pretty

File: test/fork/smoke-test/Smoke_Liquidity.sol [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+      bytes32[] memory _priceUpdateCalldata,
+      bytes32[] memory _publishTimeUpdateCalldata
+    ) = ecoPythBuilder.build(data);
+
+    vm.prank(address(0xF1235511e36f2F4D578555218c41fe1B1B5dcc1E));
+    liquidityHandler.executeOrder(_latestOrderIndex, payable(ALICE), _priceUpdateCalldata, _publishTimeUpdateCalldata, _minPublishTime, keccak256("someEncodedVaas"));
+
+    assertApproxEqRel(
+      hlpStaking.calculateShare(address(this), address(ALICE)),
+      10 * hlp.totalSupply() * 1e30 / calculator.getAUME30(false),
```
Comment (john-xina-p88) [inline test/fork/smoke-test/Smoke_Liquidity.sol:67]: let's cache the hlp price for better readability
```
uint256 hlpPrice = calculator.getAUME30(false) * ... / hlp.totalSupply();
```

File: test/fork/smoke-test/Smoke_Liquidity.t.sol [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+
+contract Smoke_Liquidity is Smoke_Base {
+  ITradingStaking internal hlpStaking = ITradingStaking(0xbE8f8AF5953869222eA8D39F1Be9d03766010B1C);
+
+  function setUp() public virtual override {
+    super.setUp();
+
+  }
+
+  function test_add_liquidity_fork() external {
```
Comment (john-xina-p88) [inline test/fork/smoke-test/Smoke_Liquidity.t.sol:25]: Go with the naming convention of
```suggestion
  function testCorrectness_addLiquidity() external {
```
and no need to do internal function if you don't reuse it anywhere.

File: test/fork/smoke-test/Smoke_Base.t.sol [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+  address internal constant OWNER = 0x6409ba830719cd0fE27ccB3051DF1b399C90df4a;
+  address internal constant POS_MANAGER = 0xF1235511e36f2F4D578555218c41fe1B1B5dcc1E; // set market status;
+  address internal ALICE;
+  address internal BOB;
+
+  function setUp() public virtual {
+    ALICE = makeAddr("Alice");
+    BOB = makeAddr("BOB");
+
+    vm.createSelectFork(vm.envString("ARBITRUM_ONE_FORK"), 130344667);
```
Comment (john-xina-p88) [inline test/fork/smoke-test/Smoke_Base.t.sol:63]:
```suggestion
    vm.createSelectFork(vm.envString("ARBITRUM_ONE_FORK"));
```

don't fix block here

Comment (jinlong9922) [reply test/fork/smoke-test/Smoke_Base.t.sol:63]: why resolve without fixed

Comment (chenciao8) [reply test/fork/smoke-test/Smoke_Base.t.sol]: sry. fixed.

**Pull request closed without merging**